### DISCUSSION TITLE
HelloWorldExampleTCP example build failed on ARM64 platform

### DISCUSSION THREAD

**Discussion by shdx2046** [eProsima/Fast-DDS#3273] [Unofficial support]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

The HelloWorldExampleTCP demo can be built on x86 machine. It should also be built successfully on ARM64 platform.

### Current behavior

When building on it locally on ARM64 platform (gcc version is 7.5.0 in my case), I got the error "reference to 'option' is ambiguous" .

### Steps to reproduce

Pull the latest Fast DDS repo.
Build the demos with below commands in a newly created build folder in the examples/cpp/dds path.

cmake ..
cmake --build .

### Fast DDS version/commit

2.8.1

### Platform/Architecture

Other. Please specify in Additional context section.

### Transport layer

TCPv4

### Additional context

Ubuntu 18.04 ARM64
GCC version: 7.5.0

The build error can be resolved by replacing line 30 "namespace option = eprosima::option;" with "using namespace eprosima;"

**Comment by Mario-DL** [comment]
Hi @shdx2046,

According to our [CONTRIBUTING.md](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md) guidelines, Ubuntu 18.04 with gcc 7.5.0 is not an [officially supported platform](https://github.com/eProsima/Fast-DDS/blob/master/PLATFORM_SUPPORT.md), so I am moving this issue to the proper discussion forum.
